docs/source/en/tutorials/basic_training.mdx

```diff
@@ -0,0 +1,408 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+[[open-in-colab]]
+
+# Train a diffusion model
+
+A popular application of diffusion models is unconditional image generation. The model generates an image that resembles the dataset it was trained on. Typically, the best results are obtained from finetuning a pretrained model on a specific dataset. You can find many of these checkpoints on the [Hub](https://huggingface.co/models?library=diffusers&sort=downloads), but if you can't find one you like, you can always train your own!
```

Comment:
Most of them would be text-to-image models (especially when sorted by downloads). Maybe we should skip this mention altogether (or stress that unconditional image generation is a "step before" text-to-image models like Stable Diffusion).

Reply:
Thanks to the new full-text search, we can find [checkpoints](https://huggingface.co/search/full-text?q=unconditional-image-generation&type=model) that include `tag: unconditional-image-generation` in the model card!